### ISSUE TITLE
Debug mode

### ISSUE THREAD

**Issue by deniak** [w3c/specberus#223]
@sideshowbarker mentioned it would be good to have a debug/verbose mode. Each time a rule fails, we have to dig into the code to check what specberus is looking for.
It would be good to display that in the UI as the old pubrules does.

**Comment by tripu** [comment]
Good idea. Perhaps in this new mode we could dump the JS trace every time an `error` is caught in the class `Specberus`&hellip;
